lib/addressfinder.rb

```diff
@@ -33,7 +34,11 @@ def cleanse(args={}) # We are keeping this method for backward compatibility
     end
 
     def verification(args={})
-      AddressFinder::Verification.new(args.merge(http: AddressFinder::HTTP.new(configuration))).perform.result
+      if configuration.verification_version.downcase == "v2"
```

Comment:
just in case?

```suggestion
      if configuration.verification_version&.downcase == "v2"
```

Reply:
Can't hurt, but unnecessary since `lib/addressfinder/configuration.rb` sets a default of `v1`